packages/react-chat/src/styles/color.ts

```diff
@@ -12,6 +12,9 @@ export const SHADOWS = {
   shadow8: 'rgba(0,0,0,0.08)',
   shadow12: 'rgba(0,0,0,0.12)',
   shadow16: 'rgba(0,0,0,0.16)',
+
+  surfaceZ1Light:
```

Comment:
Reference to V2 designs:
<img width="789" alt="Screenshot 2023-10-30 at 12 52 40 PM" src="https://github.com/voiceflow/react-chat/assets/5643574/7ce37613-5770-465c-822e-94053c710bed">